src/main/java/mate/academy/exception/RegistrationException.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.exception;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException() {

```

Comment:
Please, check in your implementation if you use all these constructors. If not, remove it